package.json

```diff
@@ -3,23 +3,32 @@
   "version": "2.0.2-0",
   "description": "An opinionated, isomorphic HTTP client.",
   "license": "BSD-3-Clause",
-  "main": "index.js",
+  "type": "module",
+  "main": "./dist/cjs/index.cjs",
+  "exports": {
+    "require": "./dist/cjs/index.cjs",
+    "import": "./lib/index.js"
+  },
   "scripts": {
+    "rollup": "rollup -c rollup.config.js",
+    "build": "npm run clear && npm run rollup",
+    "clear": "rimraf dist/ && mkdir dist",
+    "prepare": "npm run build",
+    "rebuild": "npm run clear && npm run build",
     "test": "npm run test-node",
-    "test-node": "cross-env NODE_ENV=test mocha -r esm --preserve-symlinks -t 30000 -A -R ${REPORTER:-spec} --require tests/test-mocha.js tests/*.spec.js",
-    "test-karma": "karma start karma.conf.js",
-    "test-watch": "cross-env NODE_ENV=test mocha -r esm --watch --preserve-symlinks -t 30000 -A -R ${REPORTER:-spec} --require tests/test-mocha.js tests/*.spec.js",
+    "test-node": "cross-env NODE_ENV=test mocha --preserve-symlinks -t 30000 -A -R ${REPORTER:-spec} --require tests/test-mocha.js tests/*.spec.js",
+    "test-karma": "karma start karma.conf.cjs",
+    "test-watch": "cross-env NODE_ENV=test mocha --watch --parallel --preserve-symlinks -t 30000 -A -R ${REPORTER:-spec} --require tests/test-mocha.js tests/*.spec.js",
     "coverage": "cross-env NODE_ENV=test nyc --reporter=lcov --reporter=text-summary npm run test-node",
     "coverage-ci": "cross-env NODE_ENV=test nyc --reporter=lcovonly npm run test-node",
     "coverage-report": "nyc report",
```

Comment:
Use `c8` instead of `nyc` as our other packages do now so we can keep things consistent.

Reply:
oh yeah, just noticed that coverage is broken. will update to. use c8, thanks

Reply:
Updated [747ee54](https://github.com/digitalbazaar/http-client/pull/21/commits/747ee54cc2a9e0b477750a8f5018f2a171e7df08)